simple/stats/data.py

```diff
@@ -437,3 +437,10 @@ def from_json(json: dict) -> Self:
                         measured_properties=set(
                             json.get("measuredProperties", [])),
                         verticals=set(json.get("verticals", [])))
+
+
+@dataclass
+class StatVarHierarchyResult:
+  svg_triples: list[Triple]
+  # dict[parent svg id, dict[child svg id, child specialized name]]
+  svg_specialized_names: dict[str, dict[str, str]]
```

Comment:
nit: child SVG-id to specialized name need to be a dict?

Reply:
For readability, consider type names (like typedefs):

e.g.,
```
SVGSpecializedName = tuple[str, str]
ParentSVG2Children = dict[str, SVGSpecializedName]
```

Reply:
On tuple: It needs to be a dict since there could be multiple children with specialized names.

On type aliases: +1 for readability. Done.